api/prisma/seed-staging.ts

```diff
@@ -355,6 +355,9 @@ export const stagingSeed = async (
   await prismaClient.amiChart.create({
     data: amiChartFactory(8, lakeviewJurisdiction.id, 2),
   });
+  await prismaClient.amiChart.create({
```

Comment:
suggestion: I wonder if we should update the seeded AMI chart names so it's easy to tell what jurisdiction they are from. Maybe all start with the jurisdiction name followed by a random word?